lib/puppet/provider/ios_snmp_trap/command.yaml

```diff
@@ -0,0 +1,32 @@
+---
+get_values:
+  default: 'show running-config | begin interface'
+get_instances:
+  default: '\\ninterface (?:(?:.| |\\n )*\\n)'
+delete_command_default:
+  default: 'default interface <name>'
+delete_command_no:
+  default: 'no interface <name>'
+attributes:
+  name:
+    default:
+      print_key: 'true'
+      get_value: '^.*interface (?<name>\S*)\n'
+  mac_notification_added:
```

Comment:
`mac-notification` does not appear to be supported on all our devices, the `4507r` does not have the feature it appears - so might need to document this; there may be others.

Reply:
checking